import_busstops.py

```diff
@@ -7,6 +7,7 @@ class BusImportCommand(CSVImportCommand):
 
     def __init__(self, *kargs, **kwargs):
         self.dry_run = kwargs.pop('dry_run')
+        kwargs['num_expected_records'] = 430525
```

Comment:
As long as it is roughly right, it will give an idea of timings. That's all that this is for.